include/hdlConvertor/hdlAst/hdlTypes.h

```diff
@@ -51,6 +52,19 @@ class HdlClassDef: public virtual iHdlTypeDec {
 	}
 };
 
+class HdlPhysicalDef: public virtual iHdlTypeDec {
+public:
+    std::unique_ptr<iHdlExprItem> range;
+    std::vector<
+        std::pair<std::unique_ptr<std::string>, 
```

Comment:
Is there any specific case where the name can be null? If not we can use just `std::string` instead of `std::unique_ptr<>`.